APMrover2/AP_MotorsUGV.cpp

```diff
@@ -322,8 +324,8 @@ void AP_MotorsUGV::output_throttle(SRV_Channel::Aux_servo_function_t function, f
                 // do nothing
                 break;
         }
-        // brushed-with-relay motors should receive positive values
-        throttle = fabsf(throttle);
+        // invert the ouput to always have positive value calculated by calc_pwm
+        throttle = reverse_multiplier * fabsf(throttle);
```

Comment:
So reversing the brushed motor output has two impacts?

- sets relay in the reverse direction
- changes direction of output

I'm surprised there are two changes.